webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -94,18 +97,31 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
         this.props.close();
     };
 
-    deleteChannelSubscription = (e) => {
+    deleteChannelSubscription = () => {
         if (this.props.selectedSubscription) {
             this.props.deleteChannelSubscription(this.props.selectedSubscription).then((res) => {
                 if (res.error) {
                     this.setState({error: res.error.message});
                 } else {
-                    this.handleClose(e);
+                    this.handleClose();
                 }
             });
         }
     };
 
+    handleDeactivateCancel = () => {
```

Comment:
```suggestion
    handleCancelDelete = () => {
```

Reply:
Not sure if `handleCancelDelete` or `handleDeleteCancel` sounds better. Just need to be consistent with the `handleConfirmDelete` or `handleDeleteConfirm` name.

Reply:
I think `handleCancelDelete` is fitting.  Thanks for all the reviews this morning! I'm ready to get the feature branch merged to master soon!